### ISSUE TITLE
Dependency React missing.

### ISSUE THREAD

**Issue by supra28** [siddharthkp/card-vibes#1]
React is missing in package.json is that intentional? 
Would it cause any problem for someone contributing?

**Comment by siddharthkp** [comment]
I guess it should be in peer dependencies

Well, right now there isn't a playground to even test it out, so idk how easy is it to contribute

**Comment by supra28** [comment]
There is another problem I guess, the event listeners are not getting detached as there's no call to `destroy()` .
I think it should be added to componentWillUnmount ? 
In both the repositories, here and react-vanilla-tilt.

**Comment by siddharthkp** [comment]
Yeah, if there are a lot of cards, you might end up with a memory leak

Profiling http://awesometalks.party might be a good place to start

@SaraVieira heads up!

Feel free to create a pull request if there's a problem 😄

**Comment by supra28** [comment]
The version used in  [ http://awesometalks.party](http://awesometalks.party) doesn't have the updated build after querySelectorAll was replaced by refs (Because the the project wasn't built again after the update). I've sent a [pull request](https://github.com/siddharthkp/card-vibes/pull/3) for that in this repository,
